GVRf/Framework/src/org/gearvrf/GVRScene.java

```diff
@@ -29,22 +29,30 @@
     public GVRScene(GVRContext gvrContext) {
         super(gvrContext, NativeScene.ctor());
 
+        GVRCamera centerCamera = new GVRPerspectiveCamera(gvrContext);
```

Comment:
Do we really want to always have a Center Camera? Would it make sense to lazy-create one, if and only if the app does a `captureScreenCenter()`?